lib/scss-preprocessor/broccoli-import-theme-components.js

```diff
@@ -40,4 +49,23 @@ Import.prototype.addImports = function(content) {
   return content + importsString;
 };
 
+Import.prototype.addThemeComponentClasses = function(content) {
+  var inputPath = this.inputPaths[0];
+  var importFiles = walkSync(inputPath, { globs: this.include });
```

Comment:
Current plan is to use `themeCore` to get the component names from the theme and parentTheme